app/src/main/java/org/mozilla/fenix/gleanplum/MessagesManager.kt

```diff
@@ -0,0 +1,83 @@
+package org.mozilla.fenix.gleanplum
+
+import android.content.Context
+import org.mozilla.experiments.nimbus.internal.FeatureHolder
+import org.mozilla.fenix.nimbus.Messaging
+import org.mozilla.fenix.nimbus.StyleData
+import java.util.SortedSet
+import kotlin.Comparator
+
+class MessagesManager(
+    private val context: Context,
+    private val storage: MessageStorage,
+    private val messagingFeature: FeatureHolder<Messaging>
+) : MessageController {
+
+    private val availableMessages: SortedSet<Message> = sortedSetOf(
+        Comparator { message1, message2 -> message2.style.priority.compareTo(message1.style.priority) }
```

Comment:
@jhugman apart from the priority, is there anything else that will help us to identify which messages should be shown first? I was thinking about triggers but not sure if this is out of scope for now?